### PR TITLE
use bin/rails default instead of rake commands [ci skip]

### PR DESCRIPTION
**Pull request by Gaurav2728**
I go through the `http://edgeguides.rubyonrails.org/` and found `rake` commands in various files that are in RAILS 5.0 implement by `bin/rails` command. I try to change all that can be directly use `bin/rails …`

### CODE REVIEW

File: guides/source/active_record_migrations.md [hunk 1]
```diff
@@ -809,7 +809,7 @@ Active Record believes that it has already been run.
 
 ### Running Migrations in Different Environments
 
-By default running `rake db:migrate` will run in the `development` environment.
+By default running `rake or bin/rails db:migrate` will run in the `development` environment.
```
Comment (maclover7) [inline guides/source/active_record_migrations.md]: Can we just change this to `bin/rails`

Comment (Gaurav2728) [inline guides/source/active_record_migrations.md]: @maclover7 yup !! here is my intentaion to show the various changes. If community agree to change all `rake` implemented rails commands using `bin/rails` in guides or elsewhere. i'll change all over the places. More over can directly push into docrails.

**Comment by maclover7** [comment]
@rafaelfranca @kaspth Do we want to change all of the docs to reflect the `rake` --> `bin/rails` change in 5.0? Better to change all or nothing to stay consistent.

**Comment by kaspth** [comment]
Change all the things! 🎷

**Comment by maclover7** [comment]
@Gaurav2728 can you remove the `rake` then? Thanks!

**Comment by Gaurav2728** [comment]
@maclover7  updated :100:

File: guides/source/active_record_migrations.md [hunk 2]
```diff
@@ -886,7 +886,7 @@ Occasionally you will make a mistake when writing a migration. If you have
 already run the migration then you cannot just edit the migration and run the
 migration again: Rails thinks it has already run the migration and so will do
 nothing when you run `rails db:migrate`. You must rollback the migration (for
-example with `rake db:rollback`), edit your migration and then run
+example with `rails db:rollback`), edit your migration and then run
```
Comment (maclover7) [inline guides/source/active_record_migrations.md]: Can we change this to bin/rails to match the other ones?

Comment (Gaurav2728) [inline guides/source/active_record_migrations.md]: @maclover7  ok... give me some time, i'll go through other files as well as keep changes here. :)

**Comment by Gaurav2728** [comment]
@maclover7 @kaspth :metal:

File: guides/source/active_record_migrations.md [hunk 3]
```diff
@@ -784,7 +784,7 @@ it with the seed data.
 ### Resetting the Database
 
 The `rails db:reset` task will drop the database and set it up again. This is
-functionally equivalent to `rake db:drop db:setup`.
+functionally equivalent to `rails db:drop db:setup`.
```
Comment (maclover7) [inline guides/source/active_record_migrations.md]: bin/rails

File: guides/source/routing.md [hunk 4]
```diff
@@ -1116,7 +1116,7 @@ Rails offers facilities for inspecting and testing your routes.
 
 ### Listing Existing Routes
 
-To get a complete list of the available routes in your application, visit `http://localhost:3000/rails/info/routes` in your browser while your server is running in the **development** environment. You can also execute the `rake routes` command in your terminal to produce the same output.
+To get a complete list of the available routes in your application, visit `http://localhost:3000/rails/info/routes` in your browser while your server is running in the **development** environment. You can also execute the `rails routes` command in your terminal to produce the same output.
```
Comment (maclover7) [inline guides/source/routing.md]: bin/rails

File: guides/source/routing.md [hunk 5]
```diff
@@ -1125,7 +1125,7 @@ Both methods will list all of your routes, in the same order that they appear in
 * The URL pattern to match
 * The routing parameters for the route
 
-For example, here's a small section of the `rake routes` output for a RESTful route:
+For example, here's a small section of the `rails routes` output for a RESTful route:
```
Comment (maclover7) [inline guides/source/routing.md]: bin/rails

**Comment by maclover7** [comment]
:+1: Thank your for your contribution!

**Pull request merged**